projects/frontend/data-pipelines/gui/e2e/support/pages/base/data-pipelines/data-pipelines-base.po.js

```diff
@@ -364,7 +365,8 @@ export class DataPipelinesBasePO extends BasePagePO {
                     {
                         pathToFixture: `/base/data-jobs/${TEAM_VDK}/short-lived/${TEAM_VDK_DATA_JOB_TEST_V12}.json`
                     }
-                ]
+                ],
+                optional: isDeleteOptional
```

Comment:
Can you add a better description of what this flag does? It's not entirely clear from the doc.

Reply:
I just updated the plugin and method jsdoc.
Thank you for raising this 👍